policyengine_us/parameters/gov/states/oh/tax/income/deductions/deductions.yaml

```diff
@@ -11,9 +11,11 @@ values:
     - spm_unit_pell_grant
     - educator_expense
     - disability_benefits
+    - oh_unreimbursed_medical_care_expense_deduction
```

Comment:
So Ohio treats deductions as subtractions? Does it also have itemized deductions?

Reply:
Yes deductions are treated as subtractions. The below is the only reference to itemized deductions:

![image](https://github.com/PolicyEngine/policyengine-us/assets/110687043/dd6bd674-402b-4a33-ba5c-bc6b857ed9f0)

Reply:
line 22